db/db_impl/db_impl.cc

```diff
@@ -3182,6 +3182,7 @@ bool DBImpl::GetAggregatedIntProperty(const Slice& property,
   }
 
   uint64_t sum = 0;
+  bool ret = true;
   {
     // Needs mutex to protect the list of column families.
     InstrumentedMutexLock l(&mutex_);
```

Comment:
Seems mutex is hold to make sure column families list is not changed. Is it still true if we unlock inside of the call?

Reply:
The intent is not to prevent column family set from changing. The purpose is to use `mutex_` to protect the data structure. I think it's OK for the column family set to change in the meantime.
Given this, the usage of mutex is not necessary, according to https://github.com/facebook/rocksdb/blob/master/db/column_family.h#L616

Reply:
sorry, do you mean the mutex here should be removed?

Reply:
Oh, I see the confusion. We should hold the mutex before iterating the column family set, but we can release and acquire, as follows.
```
mutex_.Lock();
for (auto* cfd : column_family_set_) {
  cfd->Ref();
  mutex_.Unlock();
  mutex_.Lock();
  cfd->UnrefAndTryDelete();
}
```